CS4125Project/src/MenuUI/CheckoutUI.java

```diff
@@ -19,18 +21,24 @@
 
 /**
  *
- * @author hello
+ * @author Matthew
  */
 public class CheckoutUI extends javax.swing.JFrame {
 
     private ArrayList<Purchase> items = new ArrayList<Purchase>();
+    private final LocalizationDB localizationDB;
     
     /**
      * Creates new form CheckoutUI
      */
     public CheckoutUI() {
         initComponents();
         this.setDefaultCloseOperation(JFrame.HIDE_ON_CLOSE);
+        
+        localizationDB = DBControler.getInstance().getLocalizationDB();
+        checkoutButton.setText(localizationDB.GetLocalization("CHECKOUT"));
```

Comment:
won't compile. Did you change the name of the method in the class without changing its name wherever it was used?